Dockerfile.tests

```diff
@@ -0,0 +1,17 @@
+FROM registry.devshift.net/bayesian/bayesian-api
+MAINTAINER Tomas Tomecek <ttomecek@redhat.com>
```

Comment:
Why is this dockefile needed? It looks like it was copied from worker, same case as in Dockerfile.migrations.

Reply:
I can remove this and the one with migrations.